entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/mapper/TypeToVariantAsClassNameMapper.kt

```diff
@@ -27,7 +33,7 @@ object TypeToVariantAsClassNameMapper {
             "PoolVector3Array" -> ClassName("godot.core.Variant.Type", "POOL_VECTOR3_ARRAY")
             "PoolColorArray" -> ClassName("godot.core.Variant.Type", "POOL_COLOR_ARRAY")
             else -> {
-                if (typeAsString.startsWith("VariantArray<")) {
+                if (type?.isCompatibleList() == true) {
```

Comment:
```suggestion
                if (type?.isCompatibleList()) {
```

Reply:
`type?.isCompatibleList()` returns a `Boolean?` but for an if statement you need a `Boolean`
So if `type?.isCompatibleList()` returns `null` it's `!= true`.
Or did i not get your point?

Reply:
Ohhh, didn't realize that it returns `Boolean?`. Can `isCompatibleList` list return `Boolean` instead?

Reply:
It does but `type` can be null

Reply:
opther solution would be `type != null && type.isCompatibleList()`

Reply:
@chippmann `type != null && type.isCompatibleList()` that is better and more readable too.

Reply:
Changed it :-)